src/components/Map/index.tsx

```diff
@@ -73,18 +73,29 @@ const Map = ({
             }
 
             if (LayersControlProps && layersControlRef.current) {
-                map.addControl(new MapControl(layersControlRef.current), 'top-left');
+                // map.addControl(new MapControl(layersControlRef.current), 'top-left');
             }
 
             map.on('load', () => {
                 onLoad(map);
             });
             mapRef.current = map;
         }
+        return () => {
```

Comment:
Isn't this clean up needed? whenever we dismount the map, we should clear it out, right?

Reply:
ahh my bad. Thanks :)